### ISSUE TITLE
Teste teste

### ISSUE THREAD

**Issue by Donizeti23** [Donizeti23/Hello-World#4]
► /play trombone
Teste

**Comment by Donizeti23** [comment]
<!DOCTYPE HTML>
<html ng-app="todoApp">
<head>
    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.4/angular.min.js"></script>
</head>
<body>
    <h2>To-Do App</h2>
    <div ng-controller="todoList">
        <span>Items: {{todos.length}}</span>
        <ul>
            <li ng-repeat="todo in todos">
                <label class="checkbox">
                    <input type="checkbox" ng-model="todo.done">
                    <span class="done-{{todo.done}}">{{todo.text}}</span>
                </label>
            </li>
        </ul>
        <form ng-submit="addTodo()">
            <input type="text" ng-model="todoText" size="30"
               placeholder="add new todo here">
            <input type="submit" value="add">
        </form>
    </div>
    <script>
    var app = angular.module('todoApp', []);
    
    app.controller('todoList', function($scope) {
        $scope.todos = [ {text:'Learn AngularJS', done:true},
            {text:'build an AngularJS app', done:false}];
            
        $scope.addTodo = function() {
            if($scope.todoText !== '') {
                $scope.todos.push({text:$scope.todoText, done:false});
                $scope.todoText = '';
            }
        };
    });
    </script>
</body>
</html>
